fingerprint/capture/src/main/java/com/simprints/fingerprint/capture/screen/ObserveFingerprintScanStatusUseCase.kt

```diff
@@ -65,7 +65,13 @@ class ObserveFingerprintScanStatusUseCase @Inject constructor(
 
     private suspend fun turnFlashingLedsOn() {
         if (ledsMode == VISUAL_SCAN_FEEDBACK) {
-            scannerManager.scanner.turnFlashingOrangeLeds()
+            scannerManager.scanner.turnFlashingWhiteLeds()
```

Comment:
Just for better consistency and clarity, a naming suggestion:
```suggestion
            scannerManager.scanner.turnOnFlashingWhiteSmileLeds()
```

Reply:
Done Thanks @alex-vt